config/user.js

```diff
@@ -1,7 +1,9 @@
 const CERTIFICATE_DEVSERIAL = '00';
-
+const LOCAL_URL = 'exp://192.168.0.155'; //use you local IP
```

Comment:
Any problems with using information from `Expo.Constants.linkingUri` for `LOCAL_URL` and `MOBILE_APP_URL` ? I certainly don't want any of these defines here. Use `PRODUCTION_URL = `grapql-app://`` please.

Reply:
I can't import Expo in this file. I import Expo to the server side, but I received an error message 

```
/home/sergey/development/apollo-universal-starter-kit/node_modules/expo/src/Expo.js:2
import './environment/validate';
^^^^^^

SyntaxError: Unexpected token import
```

Reply:
I didn't mean to use `Expo.Constants.linkingUri` in this file, but rather in the source code for mobile platforms instead of `LOCAL_URL` and `MOBILE_APP_URL` settings. `LOCAL_URL` and `MOBILE_APP_URL` settings should be removed from here